test/kitchen/test/integration/common/rspec_datadog/spec_helper.rb

```diff
@@ -207,6 +207,26 @@ def restart(flavor)
   result
 end
 
+def log_trace(flavor, action)
+  service = get_service_name(flavor)
+  if os == :windows
+    if action == "stop" || action == "restart"
+      system "wevtutil qe System /q:\"*[System[(EventID=7040) and (EventData[Data[@Name='param1']='#{service}'])]\""
+    end
+    if action == "start" || action == "restart"
+      system "wevtutil qe System /q:\"*[System[(EventID=7036) and (EventData[Data[@Name='param1']='#{service}'])]\""
+    end
+  else
```

Comment:
I'm not sure if I should `grep` the service action (stop/start/restart) in the below commands. I chose to display the maximum